examples/sink_video.exs

```diff
@@ -0,0 +1,62 @@
+# Before running this example, make sure that target RTMP server is live.
+# If you are streaming to eg. Youtube, you don't need to worry about it.
+# If you want to test it locally, you can run the FFmpeg server with:
+# ffmpeg -y -listen 1 -f flv -i rtmp://localhost:1935 -c copy dest.flv
+
+Mix.install([
+  :membrane_realtimer_plugin,
+  :membrane_hackney_plugin,
+  {:membrane_rtmp_plugin, path: __DIR__ |> Path.join("..") |> Path.expand()}
+])
+
+defmodule Example do
+  use Membrane.Pipeline
+
+  @samples_url "https://raw.githubusercontent.com/membraneframework/static/gh-pages/samples/big-buck-bunny/bun33s_480x270.h264"
+
+  @impl true
+  def handle_init(_ctx, destination: destination) do
+    structure = [
+      child(:video_source, %Membrane.Hackney.Source{
+        location: @video_url,
+        hackney_opts: [follow_redirect: true]
+      })
+      |> child(:video_parser, %Membrane.H264.FFmpeg.Parser{
```

Comment:
Don't we want to use elixir parser in most places at the moment?

Reply:
I am just worried how crucial is the `skip_until_keyframe?` option in that case

Reply:
I guess that it would work fine when using files as input, since there is probably keyframe at the beginning. Problems could arise when connecting in a middle of a stream.

Reply:
Indeed, but we cannot assume that the stream we receive in RTMP Sink is a stream from a file (despite the fact that in all the tests we are using only streams from the file)